### ISSUE TITLE
"in this collection" vs "all collections" dropdown is not visible when zoomed in

### ISSUE THREAD

**Issue by hackartisan** [pulibrary/pulfalight#1759]
refs #1717 
[full list of issues for audit1717](https://github.com/pulibrary/pulfalight/issues?q=is%3Aissue%20state%3Aopen%20audit1717)

### Location of problem on site

Collection page, e.g. https://findingaids.princeton.edu/catalog/C0178

### Description of error / warning

When the width is set to 1280px and zoom level is set to 240% the visual indicator of whether you are searching in this collection or in all collections is lost.

(tip: in firefox you can see the width of your window during resize if you have the inspect panel open - it shows at the top right of the window)

### WCAG success criterion

Not sure; this evaluation is part of the "resize content" step in the [WebAIM evaluation guide](https://webaim.org/articles/evaluationguide/)

### Recommended fix

Make sure the content is visible when zoomed up to 400%

### Tool used to find the problem

firefox

### Screenshot

<img width="1269" height="1134" alt="Image" src="https://github.com/user-attachments/assets/2836093c-a079-4b26-9438-0d274f6820ca" />
